CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ if(APPLE)
   SET(CMAKE_XCODE_SCHEME_ENABLE_GPU_FRAME_CAPTURE_MODE "Disabled")
 endif()
 
+if("${CMAKE_HOST_SYSTEM_NAME}" STREQUAL "FreeBSD")
+  set(FreeBSD YES)
```

Comment:
Can you change this to all uppercase to match the other cmake variables we have.